alarms/status.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+
+package alarms
+
+import (
+	"encoding/json"
+	"strings"
+
+	svcerr "github.com/absmach/supermq/pkg/errors/service"
+)
+
+type Status uint8
+
+const (
+	ReportedStatus Status = iota
+	AssignedStatus
+	ResolvedStatus
+	IgnoredStatus
```

Comment:
Can we have two main alarm states:
- Active
- Cleared

Additionally, we need a status to track whether an alarm is Acknowledged or Not Acknowledged.
Any user with the appropriate access should be able to acknowledge an alarm.

We should also record who acknowledged the alarm and the timestamp of acknowledgment.

Reply:
I am not sure about active and cleared states. Can you expound further?

Reply:
Alarms active means the related metrics crossed the given threshold , that might in any severity 
If comes backs to normal, then alarm is cleared 

I think for alarms other state are not required 
Alarms will be cleared automatically when the metric falls below threshold.